src/commands/data/connectors/update.ts

```diff
@@ -47,9 +75,17 @@ export default class ConnectorsUpdate extends BaseCommand {
 
     cli.action.start(`Updating Data Connector ${args.connector}`)
     try {
+      const {body: res} = await this.shogun.get<PostgresConnector>(
+        `/data/cdc/v0/connectors/${args.connector}`,
+        this.shogun.defaults
+      )
+      const tables = res.tables.concat(tablesToAdd).filter(t => !tablesToRemove.includes(t))
+      const excludedColumns = res.excluded_columns.concat(excludedColumnsToAdd).filter(c => !excludedColumnsToRemove.includes(c))
```

Comment:
Do we need to `uniq` the table and column lists? Or will shogun do that for us?

Reply:
Yup